pkg/cri/config/config_windows.go

```diff
@@ -23,10 +23,28 @@ import (
 
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/pkg/cri/streaming"
+	"github.com/pelletier/go-toml"
 )
 
 // DefaultConfig returns default configurations of cri plugin.
 func DefaultConfig() PluginConfig {
+	hypervisorShimOpts := `
+		# SamdboxPlatform is a CRI setting that specifies the platform
+		# architecture for all sandbox's in this runtime. Values are
+		# 'windows/amd64' and 'linux/amd64'.
+		SandboxPlatform = "windows/amd64"
+
+		# SandboxIsolation is a CRI setting that specifies the isolation level of
+		# the sandbox. For Windows runtime PROCESS and HYPERVISOR are valid. For
+		# LCOW only HYPERVISOR is valid and default if omitted.
```

Comment:
These comments probably need tweaking. For instance it mentions LCOW which isn't supported through CRI at all today.

Reply:
Was curious if we'd wanna tweak this or just pull straight from what's present in the shim options. Probably makes sense to tweak these for now

Reply:
Yeah I think we definitely want to make these descriptions make sense in the context of showing up in a containerd config, rather than the original definitions in hcsshim.

Reply:
Changed the comments up, ptal